src/test8/java/test/defaultmethods/DefaultMethodTest.java

```diff
@@ -0,0 +1,86 @@
+package test.defaultmethods;
+
+import org.testng.Assert;
+import org.testng.ITestClass;
+import org.testng.ITestNGListener;
+import org.testng.ITestNGMethod;
+import org.testng.TestNG;
+import org.testng.annotations.Test;
+import test.SimpleBaseTest;
+import test.listeners.TestAndClassListener;
+
+public class DefaultMethodTest extends SimpleBaseTest {
+
+    @Test(description = "Test default methods defined in a interface should be run")
+    public void testDefaultShouldRun() {
+        TestNG tng = create(TestA.class);
+        final TestCounter listener = new TestCounter();
```

Comment:
Please remove `final` (and those below).

Also, it looks like all these new test methods share a lot of common code, any way you can refactor?